include/xkbcommon/xkbcommon.h

```diff
@@ -303,6 +303,14 @@ typedef uint32_t xkb_led_mask_t;
 
 #define XKB_KEYCODE_MAX     (0xffffffff - 1)
 
+/**
+ * Keysyms are 32-bit integers with the 3 most significant bits always set
+ * to zero. See: Appendix A “KEYSYM Encoding” of the X Window System Protocol.
+ *
+ * @since 1.6.0
+ */
+#define XKB_KEYSYM_MAX      (0x1fffffff)
```

Comment:
aside from consistency with the ones above but - why is this in parenthesis? 

`XKB_KEYCODE_MAX` needs it (though it would be trivial to just specify it as the actual numeric value instead with a comment for the humand-readable one but here? I'm struggling to think of where the parens make a difference.

Reply:
I used parentheses just for consistency with the other ones. Since the new constants has no special requirement, let’s drop them.